Makefile

```diff
@@ -0,0 +1,16 @@
+all: elastic_search_up index_documents give_indexing_some_time rest_api_up run_query
+
+elastic_search_up:
+	docker-compose up elasticsearch
```

Comment:
shouldn't it also have the -d argument, as in docker-compose for haystack api? I think it ensures that the docker is running in the background

Reply:
ah yes, good catch! I left it out to monitor the command line, will add it back in

Reply:
https://github.com/rafaljanwojcik/board-game-rules-explainer/pull/1/commits/93110fe7754538d6c2aa360f93182698cf19236f